VKUI/add-dependabot-pr-milestone/src/isPRFromDependabotAndRelevant.ts

```diff
@@ -0,0 +1,24 @@
+import { WebhookPayload } from '@actions/github/lib/interfaces';
+
+// Проверяет, является ли PR от Dependabot и затрагивает ли нужные зависимости
+export function isPRFromDependabotAndRelevant(
+  pullRequest: Exclude<WebhookPayload['pull_request'], undefined>,
+): boolean {
+  if (pullRequest.user.login !== 'dependabot[bot]') return false;
+
+  const relevantDependencies = ['@swc/helpers', '@vkontakte/icons', '@vkontakte/vkjs', 'date-fns'];
+  const packageJsonPath = 'packages/vkui/package.json';
```

Comment:
```suggestion
  const packageJsonPath = 'packages/vkui/package.json';
  const { dependencies: relevantDependencies } = JSON.parse(fs.readFileSync('packages/vkui/package.json', 'utf8'));
  const relevantDependencies = packageJson.depene;
```

чтобы не зашивать зависимости в экшене

вообще экшен стоит сделать максимально обособленным

мы могли бы передевать через аргументы нужные нам данные, вот например как бы могли передать массив зависимостей:

```yml
      - name: Get @vkontakte/vkui dependencies
        id: dependencies
        run: echo "value=$(node -p "Object.keys(require('./packages/vkui/package.json').dependencies)")" >> $GITHUB_OUTPUT

      - name: Report lint results
        if: ${{ !cancelled() }}
        uses: VKCOM/gh-actions/VKUI/add-dependabot-pr-milestone@main
        with:
          dependencies: ${{ fromJSON(steps.dependencies.outputs.value) }}
          token: ${{ secrets.DEVTOOLS_GITHUB_TOKEN }}
```

ну и любые другие данные так можно передать

Reply:
Перенес код определения зависмостей в репозиторий VKUI